uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request-ctrl.js

```diff
@@ -22,6 +23,12 @@ define([
         vm.loading.absenceTypes = true;
         initRequest();
 
+        vm.initParams = {
+          absenceType: {
```

Comment:
its to filter in the AbsenceType.all call which is different for each leave type